Source/Diagnostics/ComputeDiagFunctors/JdispFunctor.H

```diff
@@ -0,0 +1,42 @@
+#ifndef WARPX_JDISPFUNCTOR_H_
+#define WARPX_JDISPFUNCTOR_H_
+
+#include "ComputeDiagFunctor.H"
+
+#include <AMReX_BaseFwd.H>
+
+/**
+ * \brief Functor to cell-center MF for electron current density from Kinetic-Fluid
```

Comment:
This comment should be updated since we are now calling this the displacement current.